src/features/projects/components/projectDetails/ProjectInfo.tsx

```diff
@@ -239,13 +258,44 @@ function ProjectInfo({ project }: Props): ReactElement {
       )}
 
       <div style={{ display: 'flex' }}>
-        {project?.metadata?.siteOwnerName && (
+        {project?.metadata?.siteOwnerType && (
           <div className={styles.projectMoreInfo}>
             <div className={styles.infoTitle}>
-              {t('manageProjects:siteOwnership')}
+              {t('manageProjects:siteOwnerShip')}
             </div>
-            {project.metadata?.siteOwnerType && (
-              <div className={styles.infoText} style={{ fontWeight: 'bold' }}>
+            {project?.metadata?.siteOwnerType && (
+              <div className={styles.infoText}>
+                {ownerTypes.map((ownerType: any, index: any) => {
+                  return (
+                    <React.Fragment key={ownerType}>
+                      {t(`manageProjects:${ownerType}`)}
+                      {index === ownerTypes.length - 2
+                        ? ' and '
+                        : index === ownerTypes.length - 1
+                        ? '.'
+                        : ', '}
+                    </React.Fragment>
+                  );
+                })}
+              </div>
+            )}
+
+            <div className={styles.infoText}>
```

Comment:
This is fine now, with a minor adjustment.

In scenario a> If siteOwnerName is null and acquisitionYear is not null

"Since" should start with a capital "S".

(This does not apply to scenario b.)

Reply:
Works at https://planet-webapp-584bo86bh-planetapp.vercel.app/the-million-tree-campaigns-mtc